geo-types/CHANGES.md

```diff
@@ -1,5 +1,10 @@
 # Changes
 
+## Master (not released)
```

Comment:
Yes, thank you for doing this! Tracking as we go will make the eventual release easier.